src/storm/modelchecker/results/ExplicitQualitativeCheckResult.cpp

```diff
@@ -1,4 +1,6 @@
-#include "storm/modelchecker/results/ExplicitQualitativeCheckResult.h"
+#include "storm/adapters/RationalNumberAdapter.h"  // Must come first
+
+#include "ExplicitQualitativeCheckResult.h"
```

Comment:
Please prefix this with storm/modelchecker/results as this will otherwise soon not compile anymore